ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplExternalTables.java

```diff
@@ -66,6 +67,9 @@ private ReplExternalTables(){}
 
   public static String externalTableLocation(HiveConf hiveConf, String location) throws SemanticException {
     String baseDir = hiveConf.get(HiveConf.ConfVars.REPL_EXTERNAL_TABLE_BASE_DIR.varname);
+    if (StringUtils.isEmpty(baseDir)) {
```

Comment:
At the time of load the REPL_EXTERNAL_TABLE_BASE_DIR fully qualified path is not needed?